src/components/Step1/Step1Content.tsx

```diff
@@ -1,43 +1,16 @@
 import React, { ReactElement } from 'react'
-import { InformationBox } from 'kyb-core'
-import { useSelector } from 'react-redux'
-import DynamicFormattedMessage from '../common/DynamicFormattedMessage'
-import { IStore } from '../../interfaces/Store'
+import { useStep1FormHook } from '../../hooks/useStep1FormHook'
+import Step1InfoSection from './Step1InfoSection'
+import { Step1Form } from './Step1Form'
 
 function Step1Content(): ReactElement {
-  const nom = useSelector((state: IStore) => state.folderReducer.folder.nom)
+  const step1HookData = useStep1FormHook()
+  const { existingDocuments } = step1HookData
+
   return (
     <>
-      <div className="xl:w-9/12 xl:pr-28">
-        <h3 className="subtitle mb-4">{nom}</h3>
-        <div className="text-default mb-6">
-          <DynamicFormattedMessage id="step1.text1" tag="p" />
-          <DynamicFormattedMessage id="step1.text2" tag="p" />
-        </div>
-        <InformationBox>
-          <>
-            <p className="mb-2 leading-tight">
-              <DynamicFormattedMessage
-                id="step1.information-box.text1"
-                className="font-bold"
-              />
-              <DynamicFormattedMessage id="step1.information-box.text2" />
-            </p>
-            <p className="leading-tight">
-              <DynamicFormattedMessage id="step1.information-box.text3" />
-              <DynamicFormattedMessage
-                id="step1.information-box.text4"
-                className="font-bold"
-              />
-            </p>
-          </>
-        </InformationBox>
-        <DynamicFormattedMessage
-          id="step1.text3"
-          className="font-bold my-20 text-default"
-          tag="h4"
-        />
-      </div>
+      <Step1InfoSection existingDocuments={existingDocuments} />
+      <Step1Form step1HookData={step1HookData} />
```

Comment:
```suggestion
      <Step1Form  {...step1HookData} />
```
and here src/components/Step1/Step1Form.tsx
```suggestion
      const Step1Form: FC = ( props: IStep1HookData)
```

Reply:
Done.